svelte.config.js

```diff
@@ -0,0 +1,27 @@
+import adapter from '@sveltejs/adapter-auto';
+import { vitePreprocess } from '@sveltejs/kit/vite';
+
+/** @type {import('@sveltejs/kit').Config} */
```

Comment:
Svelte is written in JS with JSDoc rather than TS, so the type annotations look like this. It should be pointing here:
https://github.com/sveltejs/kit/blob/10008b35e7194be1d9e54347731cab772b511ee4/packages/kit/src/exports/public.d.ts#L167